tasks/main.yml

```diff
@@ -71,11 +71,14 @@
     force_basic_auth: true
   register: grafana_conf_existing_datasources
 
-- name: Find available datasources
+- name: Generate list of enabled datasources
   set_fact:
-    grafana_conf_available_datasources: "{{ grafana_conf_grafana_datasources|map('extract', grafana_conf_grafana_datasource_definitions)|list }}"
+    grafana_conf_enabled_datasources:
+      "{{ grafana_conf_enabled_datasources + [grafana_conf_grafana_datasource_definitions[item.key]] }}"
+  with_dict:
+    "{{ grafana_conf_grafana_datasources | default({}, True) }}"
```

Comment:
nit: Why not put grafana_conf_grafana_datasources in defaults and remove the default here?

Reply:
Done